src/mmrelay/config.py

```
@@ -729,6 +729,41 @@ def is_e2ee_enabled(config: dict[str, Any] | None) -> bool:
     return encryption_enabled or e2ee_enabled
 
 
+def _iter_readable_config_mappings(args: Any = None) -> Iterable[dict[str, Any]]:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `args`

(ANN401)